playbooks/install.py

```diff
@@ -54,8 +61,9 @@ def install_bench(args):
 	# clone bench repo
 	clone_bench_repo()
 
-	if args.develop:
-		run_playbook('develop/install.yml', sudo=True)
+	# args is namespace, but we would like to use it as dict in calling function, so use vars()
+	if args.develop or args.setup_production:
+		run_playbook('develop/install.yml', sudo=True, extra_args=vars(args))
```

Comment:
If develop, run `develop/install.yml`

else run `setup_production.yml` and include `develop/install.yml` in it